themes/default/archetypes/blog-post/index.md

```diff
@@ -41,7 +41,7 @@ Either way, avoid using images or code samples [in the first 70 words](https://g
 
 ## Writing the Post
 
-For help assembling the content of your post, see [BLOGGING.md](https://github.com/pulumi/docs/blob/master/BLOGGING.md). For general formatting guidelines, see the Style Guide section of [CONTRIBUTING.md](https://github.com/pulumi/docs/blob/master/CONTRIBUTING.md#style-guide).
+For help assembling the content of your post, see [BLOGGING.md](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md). For general formatting guidelines, see the Style Guide section of [CONTRIBUTING.md](https://https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
```

Comment:
```suggestion
For help assembling the content of your post, see [BLOGGING.md](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md). For general formatting guidelines, see the [Style Guide section of CONTRIBUTING.md](https://https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
```

small thing that makes this link text a bit more descriptive. or maybe just Style Guide could be the link text.